### PR TITLE
Add Korean translation for Documentation

### PR DESCRIPTION
**Pull request by kyeongwan**
Hi!
I use Quick very well. So I added Korean document for Korean user!

Please review.

Thanks,
kyeongwan.

### CODE REVIEW

File: Documentation/Ko-kr/BehavioralTesting.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+```
+
+동작 테스트를 작성할 때 중요한 포인트 :
+
+- 정확히 이 애플리케이션 코드가 무엇을 해야 하는지?
+
+- 테스트가 *오직* 동작만을 테스트하는가?
+
+  또는 코드가 다르게 작동함에 따라 실패할 수 있는가?
```
Comment (modocache) [inline Documentation/Ko-kr/BehavioralTesting.md]: I think you're missing a bullet point here.

File: Documentation/Ko-kr/ArrangeActAssert.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+새로운 이름:
+
+1. 무엇을 테스트하고 있는지 명확합니다 : `testPeel` 은  `Banana.peel()` 메소드 임을 나타냅니다.
+2. 어떤 때 테스트가 통과하거나 실패하는지 명확합니다 : `makesTheBananaEdible` 는 메소드가 호출되면 바나나가 식용 가능한 상태임을 나타냅니다.
+
+## 테스트 시 조건들
+
+`offer()` 함수를 사용해서 사람들에게 바나나를 제공하고 싶다고 가정해 보겠습니다 :
+
+```swift
```
Comment (modocache) [inline Documentation/Ko-kr/ArrangeActAssert.md]: In the other languages, this is \`\`\`diff, not \`\`\`swift, which makes GitHub highlight the added/removed lines a little better.

File: Documentation/Ko-kr/QuickExamplesAndGroups.md [hunk 3]
```diff
@@ -0,0 +1,462 @@
+# Quick Example과 Example그룹으로 구성된 테스트
+
+Quick에는 **examples**와 **example groups**를 정의하기 위한 특별한 문법이 있습니다.
+
+*[XCTest를 사용한 효과적인 테스트 : Arrange, Act, 그리고 Assert](ArrangeActAssert.md)*에서,
```
Comment (modocache) [inline Documentation/Ko-kr/QuickExamplesAndGroups.md]: It looks like this isn't being italicized properly. In en-us, this is italicized as _Effective Tests Using XCTest: Arrange, Act, and Assert_. But in this ko-kr version, it shows up as \*XCTest를 사용한 효과적인 테스트 : Arrange, Act, 그리고 Assert\*. Any idea why the \*\* isn't being transformed into italics?

Comment (kyeongwan) [reply Documentation/Ko-kr/QuickExamplesAndGroups.md]: I used a program called Typora for my convenience.
But it seems to have broken a little grammar.
I'll fix it soon!

**Comment by kyeongwan** [comment]
I've already received feedback from people around the developer community in Korea!

**Comment by modocache** [comment]
Great!

Please comment once you're done addressing my previous comments, I don't want to miss this. I think you're still missing one of the bullet points I mentioned, for example.

**Comment by kyeongwan** [comment]
I think I have already done it!

This is the original text.
- What exactly should this application code do?
- Is my test verifying *only* that behavior? Or could it fail due to other aspects of how the code works?

And this is a translation


- 정확히 이 애플리케이션 코드가 무엇을 해야 하는지?
- 테스트가 *오직* 동작만을 테스트하는가? 또는 코드가 다르게 작동함에 따라 실패할 수 있는가?

In previous commit I made a mistake breaking a new line!
```diff
+동작 테스트를 작성할 때 중요한 포인트 :
+
+- 정확히 이 애플리케이션 코드가 무엇을 해야 하는지?
+
+- 테스트가 *오직* 동작만을 테스트하는가?
+
+  또는 코드가 다르게 작동함에 따라 실패할 수 있는가?
```
And I did this.

```diff
 - 정확히 이 애플리케이션 코드가 무엇을 해야 하는지?
-
 - 테스트가 *오직* 동작만을 테스트하는가?
-
   또는 코드가 다르게 작동함에 따라 실패할 수 있는가?
```

Please confirm!

**Pull request merged**